CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
-- logging: Original context deadline if available is logged under `deadline` field.
+- logging: Original context deadline if available is logged under `ttl` field.
```

Comment:
What prompted you to change this? I left an unnecessary comment on #2186 :)

What about `timeout`? 

I'm basing this on:
- https://grpc.io/blog/deadlines/ suggests `timeout`. 
- https://www.ibm.com/docs/sl/nsm/61.1?topic=guidelines-packet-ttl-timeout-values - which suggests `ttl` is an int describing the number of hops left. `timeout` seems to be time interval.

Reply:
Yes, probably a better name, I updated to `timeout`